.github/workflows/e2e.yml

```diff
@@ -12,7 +12,7 @@ on:
       - "*"
 
 env:
-  GO_VERSION: "1.20.8"
+  GO_VERSION: "1.20.10"
```

Comment:
Good call to bump this version 🙏 

Could you also set it in `abi_bindings_checker.yml`?

Reply:
Updated the version